MBINCompiler/Source/Commands/Convert.cs

```diff
@@ -310,22 +320,26 @@ private static void FileIsUnused(string filePath, MBINFile mbin)
         /// how to handle the file.
         /// </summary>
         /// <param name="file">The file path to check.</param>
-        /// <returns>A FileMode enum.</returns>
-        private static FileMode GetFileMode( string file ) {
+        /// <param name="fileMode">When the methord returns, contains the file mode.</param>
```

Comment:
typo ("methord")

Reply:
I was unsure about what how to describe the return value. For methods in this form the return value description always seems to be in the form of "true if <success>, otherwise false" where <success> is a description of what is considered a success. But in this case I have a specific failure condition. That being when the file exists and we don't want to overwrite it. When that happens the file mode is CreateNew and if you were to try and open the file with that file mode you would get an IOException.